### DISCUSSION TITLE
README

### DISCUSSION THREAD

**Discussion by elalish** [google/model-viewer#2072] [General]
Welcome to `<model-viewer>` discussions! We found that most of the issues being generated were actually questions on how to use this web component, so this forum is designed to make the answers more discoverable. Please search here before starting a new discussion or issue to see if there's already an answer. Note that few people mark discussions as "Answered" (if you do, thank you!) but rest assured that most of the discussions here have in fact been answered, even if the check mark isn't green.

Please file feature requests under Ideas, and please share any cool things you've done with `<model-viewer>` in Show and Tell. We like to have projects to tweet about.

If you have found a bug, please file an issue only if you can fill out the entire issue template. If you're not sure, feel free to start a discussion first. We'll let you know if an issue should be filed. Please do not post the same thing in multiple places; it makes it much harder to keep track of the conversation and will definitely **not** result in faster responses. 

Thank you, and enjoy `<model-viewer>`!

**Comment by M-El-Shayeb** [comment]
![unnamed-file-1789](https://github.com/user-attachments/assets/96b2aa5a-9b8c-451e-b676-f6aac14fd134)
